### DISCUSSION TITLE
Change cursor size

### DISCUSSION THREAD

**Discussion by Gregor-Gottschewski** [AlmasB/FXGL#1281] [Q&A]
Hi there,

I created a 256x256 px mouse cursor for my game and added it via: `gameSettings.setDefaultCursor(new CursorInfo("cursor.png", 0, 0));`. However, the cursor is shown way too small, like a dot. Is there an option to increase the cursor size?

**Edit:** Also the default FXGL cursor is small like a dot.

**Comment by Gregor-Gottschewski** [comment]
I think it's a JavaFX fault. I have the same problem when I change the cursor in a default JavaFX application.

**Reply by AlmasB** [reply]
Yes, sounds like it. The FXGL's implementation of set cursor just calls the JavaFX one. If the issue is still there, you might want to contact the JavaFX team.
